app/styles/_library/_nypr-m-play-button.scss

```diff
@@ -20,13 +20,13 @@
     .play-icon-shapes .outer-circle-fx {
       display: block;
       animation: pulse-rings-outer 5s 1s infinite;
-      fill: #1FABF1;
+      fill: #a70b0b;
```

Comment:
I believe this should be a theme level change so it doesn't change the white label/WQXR play button colors.